Tests/Spreadsheet/test-spreadsheet.cpp

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2022, Ali Mohammad Pur <mpfard@serenityos.org>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#include <LibJS/Runtime/Map.h>
+#include <LibTest/JavaScriptTestRunner.h>
+
+TEST_ROOT("Userland/Applications/Spreadsheet/Tests");
+
+#ifdef __serenity__
+static constexpr auto s_spreadsheet_runtime_path = "/res/js/Spreadsheet/runtime.js"sv;
+#else
+static constexpr auto s_spreadsheet_runtime_path = "../../../../Base/res/js/Spreadsheet/runtime.js"sv;
+#endif
+
+TESTJS_RUN_FILE_FUNCTION(String const&, JS::Interpreter& interpreter, JS::ExecutionContext& context)
```

Comment:
This should also be named `global_execution_context`, a random `context` isn't very clear.